api/model/src/testUriCompliance/java/org/projectnessie/model/TestElementsEncoding.java

```diff
@@ -495,4 +508,38 @@ static List<List<String>> randomRoundTrips() {
     }
     return r;
   }
+
+  @ParameterizedTest
+  @MethodSource
+  void allCharacters(String element) {
```

Comment:
A very "paranoid" test that exercises all possible `char`s starting with ASCII 32.